docker/Dockerfile

```diff
@@ -12,6 +12,8 @@ RUN apt-get update && apt-get install -y \
     pkg-config \
     python3 \
     python3-dev \
+    python3-pip \
+    && pip3 install --upgrade pip \
```

Comment:
@yaoyinnan 
I'm not a python expert by any means, just curious if the next line be adjusted to `pip3 install pyarrow`?

Reply:
@etolbakov Sounds reasonable.